### PR TITLE
Automated plotting

### PR DESCRIPTION
**Pull request by NicholasCowie**
# Summary
This is a tool to help plot your posterior draws. It's fairly basic at the moment and doesn't spit out the prettiest plots but is 99% automated only requiring the following inputs which are noted in the documentation:
* TOML file directory (relative to Maud home directory)
* CSV file directory (relative to Maud home directory)
* model name
* csv timestamp code

# Output Plots
The current output plots are:
* concentrations
* fluxes
* enzyme concentrations
* formation energies
* Keqs
* kcats
* Kms

# Commit History
* feat: automated plotting of population
* style: fixing black style guide
* Docs: update dependencies for plotting using plotnine

# Future Directions with Plotting
There are several features not implemented that will be once a finalised file structure is resolved:
* overlaying measured/prior values with CIs
* prettier plots in general

### CODE REVIEW

**Comment by NicholasCowie** [comment]
Can someone try this on their computer?

File: src/maud/plot_samples.py [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+import arviz as az
+import plotnine as p9
+
+from maud import io
+
+
+HERE = os.path.dirname(os.path.abspath(__file__))
+HOME = os.path.join(os.path.join(HERE, ".."), "..")
+model_name = "linear"  # toml name of file
+run_number = "202010091734"  # number specific to each run
```
Comment (teddygroves) [inline src/maud/plot_samples.py:30]: capitalise

File: src/maud/plot_samples.py [hunk 2]
```diff
@@ -0,0 +1,155 @@
...
+PATHS = {
+    "DATA": os.path.join(
+        HOME, "test/data"
+    ),  # Where the input toml file is stored, relative to the Maud folder
+    "RESULTS": os.path.join(
```
Comment (teddygroves) [inline src/maud/plot_samples.py:35]: "data" and "results" should be lower case here I think

**Review by teddygroves — CHANGES_REQUESTED** [review]
I think since `plot_samples.py` is a script it shouldn't go in the `src/maud` folder but either in the project root or in a separate `scripts` folder. 

Otherwise looks good!

**Review by teddygroves — APPROVED** [review]

**Pull request merged**